src/components/authorization/authorization.service.ts

```diff
@@ -254,6 +265,23 @@ export class AuthorizationService {
     }
   }
 
+  isSensitivityAllowed(
+    grant: Partial<
+      Record<Action, boolean> & Record<'sensitivityAccess', Sensitivity>
+    >,
+    sensitivity?: Sensitivity
+  ): boolean {
+    const sensitivityRank = { High: 3, Medium: 2, Low: 1 };
+    if (
+      sensitivity &&
```

Comment:
Should we throw an error if sensitivity check is required but none was given?

Reply:
done